track_oracle/example/track_reader_example.cxx

```diff
@@ -197,18 +204,38 @@ int load_tracks( const string& track_fn, track_handle_list_type& tracks, const s
   map< field_handle_type, size_t > track_stats, frame_stats;
   size_t frame_count = 0;
 
+  vul_timer timer;
   for (size_t i=0; i<tracks.size(); ++i)
   {
     vector< field_handle_type > ts = track_oracle_core::fields_at_row( tracks[i].row );
     for (size_t t=0; t<ts.size(); ++t) ++track_stats[ ts[t] ];
 
     frame_handle_list_type frames = track_oracle_core::get_frames( tracks[i] );
     frame_count += frames.size();
+    vector< oracle_entry_handle_type > frame_handles( frames.size() );
+    for (auto j=0; j<frames.size(); ++j)
+    {
+      frame_handles[j] = frames[j].row;
+    }
+    /*
     for (size_t j=0; j<frames.size(); ++j)
```

Comment:
Can we ditch dead code?

Reply:
Yes; I promise to do this in the followup dev/kpf-track-oracle-activities branch. :)